### PR TITLE
feat: Review UI of Navigation drawer - MEED-6831 - Meeds-io/MIPs#137

### PR DESCRIPTION
**Pull request by SaraBoutej**
This PR adds UI/Ux adjustment to the site navigation drawer

### CODE REVIEW

File: layout-webapp/src/main/resources/locale/portlet/SiteNavigation_fr.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 siteNavigation.button.tooltip.label=Navigation du site
 siteNavigation.drawer.title=Navigation du site
-siteNavigation.label.btn.createNode=Cr\u00e9er un noeud
+siteNavigation.label.btn.createNode=Ajouter un item
```
Comment (boubaker) [inline layout-webapp/src/main/resources/locale/portlet/SiteNavigation_fr.properties]: Should be managed by PO on crowdin even if it's possible to do with code

```suggestion
siteNavigation.label.btn.createNode=Cr\u00e9er un noeud
```

File: layout-webapp/src/main/resources/locale/portlet/SiteNavigation_en.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 siteNavigation.button.tooltip.label=Site navigation
 siteNavigation.drawer.title=Site navigation
-siteNavigation.label.btn.createNode=Create a node
+siteNavigation.label.btn.createNode=Add Item
```
Comment (boubaker) [inline layout-webapp/src/main/resources/locale/portlet/SiteNavigation_en.properties]: Should be managed by PO on crowdin even if it's possible to do with code

```suggestion
siteNavigation.label.btn.createNode=Create a node
```

File: layout-webapp/src/main/resources/locale/portlet/SiteNavigation_fr.properties (hunk 1 above)
Comment (SaraBoutej) [reply layout-webapp/src/main/resources/locale/portlet/SiteNavigation_fr.properties]: removed! 
cc @srenault-meeds FYI

File: layout-webapp/src/main/resources/locale/portlet/SiteNavigation_en.properties (hunk 2 above)
Comment (SaraBoutej) [reply layout-webapp/src/main/resources/locale/portlet/SiteNavigation_en.properties]: removed! 
cc @srenault-meeds FYI

**Review by boubaker — APPROVED** [review]

**Pull request merged**